examples/wdio/wdio.mocha.conf.js

```diff
@@ -56,5 +57,5 @@ exports.config = {
     onComplete: function() {
         console.log('that\'s it');
     }
-
-};
\ No newline at end of file
+    
```

Comment:
Oh wait, `insert_final_newline = true` is part of the `.editorconfig`, so this is intentional I suppose.